batchflow/models/torch/base.py

```diff
@@ -518,9 +522,124 @@ def head(cls, **kwargs):
+
+            - 'sigmoid' - ``sigmoid(inputs)``
+            - 'proba' - ``softmax(inputs)``
+            - 'labels' - ``argmax(inputs)``
+            - 'softplus' - ``softplus(inputs)``
+            - callable - a user-defined operation
+
+        ops : a sequence of operations or an ordered dict
+            auxiliary operations
+
+            If dict:
+
+            - key - a prefix for each input
+            - value - a sequence of aux operations
+
+        Raises
+        ------
+        ValueError if the number of inputs does not equal to the number of prefixes
+        TypeError if inputs is not a Tensor or a sequence of Tensors
+
+        Examples
+        --------
+
+        ::
+
+            config = {
+                'output': ['proba', 'labels']
+            }
+
+        However, if one of the placeholders also has a name 'labels', then it will be lost as the model
+        will rewrite the name 'labels' with an output.
+
+        That is where a dict might be convenient::
+
+            config = {
+                'output': {'predicted': ['proba', 'labels']}
+            }
+
+        Now the output will be stored under names 'predicted_proba' and 'predicted_labels'.
+
+        For multi-output models ensure that an ordered dict is used (e.g. :class:`~collections.OrderedDict`).
+        """
+        if ops is None:
+            ops = []
+        elif not isinstance(ops, (dict, tuple, list)):
+            ops = [ops]
+        if not isinstance(ops, dict):
+            ops = {'': ops}
+
+        if not isinstance(inputs, (tuple, list)):
+            inputs = [inputs]
+
+        for i, tensor in enumerate(inputs):
+            if not isinstance(tensor, torch.Tensor):
+                raise TypeError("Network output is expected to be a Tensor, but given {}".format(type(inputs)))
+
+            prefix = [*ops.keys()][i]
+            attr_prefix = prefix + '_' if prefix else ''
+
+            self._add_output_op(tensor, predictions, 'predictions', '', **kwargs)
+            for oper in ops[prefix]:
+                self._add_output_op(tensor, oper, oper, attr_prefix, **kwargs)
+
+    def _add_output_op(self, inputs, oper, name, attr_prefix, **kwargs):
+        if oper is None:
+            self._add_output_identity(inputs, name, attr_prefix, **kwargs)
+        elif oper == 'softplus':
+            self._add_output_softplus(inputs, name, attr_prefix, **kwargs)
+        elif oper == 'sigmoid':
+            self._add_output_sigmoid(inputs, name, attr_prefix, **kwargs)
+        elif oper == 'proba':
+            self._add_output_proba(inputs, name, attr_prefix, **kwargs)
+        elif oper == 'labels':
+            self._add_output_labels(inputs, name, attr_prefix, **kwargs)
+        elif callable(oper):
+            self._add_output_callable(inputs, oper, None, attr_prefix, **kwargs)
+
+    def _add_output_identity(self, inputs, name, attr_prefix, **kwargs):
+        _ = kwargs
+        setattr(self, attr_prefix + name, inputs)
```

Comment:
I think that `store_to_attr` is excessive in `TorchModel` because it will be exactly `setattr`. In `TFModel` it also register tensors but here we needn't to do something like that.